CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+cmake_minimum_required(VERSION 3.5)
+
+set(CMAKE_CXX_STANDARD 11)
+
+set(libunivalue_major_version 1)
+set(libunivalue_minor_version 0)
+set(libunivalue_micro_version 3)
+set(libunivalue_interface_age 1)
+# If you need a modifier for the version number.
+# Normally empty, but can be used to make "fixup" releases.
+set(libunivalue_extraversion)
+
+# libtool versioning from libunivalue
+math(EXPR libunivalue_binary_age "100 * ${libunivalue_minor_version} + ${libunivalue_micro_version}")
+math(EXPR libunivalue_current "${libunivalue_binary_age} - ${libunivalue_interface_age}")
+set(libunivalue_age ${libunivalue_current})
+set(libunivalue_revision ${libunivalue_interface_age})
+set(libunivalue_version "${libunivalue_major_version}.${libunivalue_minor_version}.${libunivalue_micro_version}${libunivalue_extraversion}")
+
+project(libunivalue
+    VERSION "${libunivalue_major_version}.${libunivalue_minor_version}.${libunivalue_micro_version}"
+    HOMEPAGE_URL "http://github.com/jgarzik/univalue/"
+    LANGUAGES CXX)
```

Comment:
HOMEPAGE_URL need the latest cmake 3.12. https://cmake.org/cmake/help/v3.5/command/project.html. I think you can just delete that line.